client/__tests__/components/DashboardHeader.test.js

```diff
@@ -0,0 +1,54 @@
+/* 
+global jest $
+expect 
+*/
+import configureMockStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+import React from 'react';
```

Comment:
Do you  really need React here when you already testing a React component with this?